community/recruitment.md

```diff
@@ -11,10 +11,10 @@ We recruit UG 1st years in the team, preferably after they have spent one semest
 For more detailed description, refer [Freshers' Selection](/docs/events/freshers-selection)
 
 ## Do we accept M.Tech and Ph.D students?
-By acceptance, we mean that they will hold a number of responsibilities
+No. By acceptance, we mean that they will hold a number of responsibilities
 - Attend our meetings
 - Participate in our workshops and their logistics
 - Collaborate on projects together
 - Be a great team member
 
-However, we have been collaborating with non-KOSS members since forever. They are welcoming to join/lead our events or pitch something to us. If you want to talk about something, you can invite them to one of your meetings as well. There are no such rules.
\ No newline at end of file
+However, we have been collaborating with non-KOSS members since forever. They are welcoming to join/lead our events or pitch something to us. If you want to talk about something, you can invite them to one of your meetings as well. There are no such rules.
```

Comment:
There are two _you_s here, It could the last couple of sentences could be edited to:
> If they want to talk about something, you can invite them to one of your meetings as well. There are no such rules preventing the same.

Reply:
I modified the lines you put slightly by removing the `you` to `we`.
>If they want to talk about something, we can have them in one of your meetings as well. There are no such rules preventing the same.

It should be in 1st person, right?

Reply:
I feel the meeting we are referring here is `ours` not `theirs` , hence this small change. Also IMO removing the last part which says `There are no such rules preventing the same.` should be removed as it adds no new information to the doc, but only adds confusion.
> If they want to talk about something, we can have them in one of `our` meetings as well.

Reply:
Made those changes you told @Shankusu993 
 @xypnox once can you check and confirm the latest changes regarding this last paragraph